src/main/resources/templates/client-create.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Add client:</title>
+</head>
+<body>
+<header>
+    <nav th:replace="_components :: navigationBar"></nav>
+</header>
+<div class="client-create-from-container">
+    <form method="post" class="client-form" th:action="@{/admin/clients/create}" th:object="${clientForm}">
```

Comment:
do każdego wymaganego inputu w formularzu (czyli w tym formularzu chyba do każdego) warto dodać atrybut `required` (chyba nie ma możliwości ustawienia tego jednorazowo dla całej formatki - ale jeśli znajdziesz sposób, to chętnie poznam)

Reply:
j.w.